pyhuman/app/workflows/browse_youtube.py

```diff
@@ -0,0 +1,80 @@
+from time import sleep
+import os
+import random
+
+# from soupsieve import select
+
+from ..utility.base_workflow import BaseWorkflow
+from ..utility.webdriver_helper import WebDriverHelper
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.common.exceptions import ElementNotInteractableException
+
+WORKFLOW_NAME = 'YoutubeBrowser'
+WORKFLOW_DESCRIPTION = 'Browse Youtube'
+
+DEFAULT_INPUT_WAIT_TIME = 2
+MAX_WATCH_TIME = 150 # Maximum amount of time to watch a video, in seconds
+MAX_WAIT_TIME = 5 # Maximum amount of time to wait after searching, in seconds
+MAX_SUGGESTED_VIDEOS = 10
+
+SEARCH_LIST = 'browse_youtube.txt'
+
+def load():
+    driver = WebDriverHelper()
+    return GoogleSearch(driver=driver)
+
+
+class GoogleSearch(BaseWorkflow):
```

Comment:
I think this class name may be a holdover from a copy-paste. Probably a good idea to name it something like YoutubeBrowse or something like that.

Reply:
Oh gosh, thanks for catching that:)